contrib/python/scripts/scale_viscous_flow_law_prefactors_aspect.py

```diff
@@ -0,0 +1,287 @@
+# This script scales the prefactors terms from viscous flow laws commonly used
+# in geodynamic modeling in a manner that is internally consistent with ASPECT's
+# conventions following Dannberg et al. (2017, https://doi.org/10.1002/2017GC006944).
+
+import numpy as np
+
+# Gleason and Tullis (1995) - Wet Quartzite - Dislocation Creep
+
+pfct_gt95_wqtz_disl_rprt = 1.8e-8 # Original reported prefactor in units of MPa^-n micrometers^m
```

Comment:
Rename all variables to be more readable, for example like:
```suggestion
prefactor_gt95_wetquartz_dislocation_reported = 1.8e-8 # Original reported prefactor in units of MPa^-n micrometers^m
```